README.md

```diff
@@ -1,66 +1,59 @@
 # Web Content Accessibility Guidelines Summary
 
-This document will help you quickly get up to speed with Web Content Accessibility Guidelines 2.1 (WCAG), and avoid common accessibility-related mistakes.
+This document will help you quickly get up to speed with Web Content Accessibility Guidelines 2.1 (WCAG), avoid common accessibility-related mistakes.
```

Comment:
We might still want the `and` here for it to read nicely?

Reply:
Yep, that was a mistake.

Reply:
Fixed.